src/__test/fixtures/load-fixtures.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { readFileSync } from 'fs'
+import { JSDOM } from 'jsdom'
+import MarkdownIt from 'markdown-it/lib'
+import taskLists from '../../index'
+
+enum FixtureFiles {
+  bullet = 'bullet.md',
+  ordered = 'ordered.md',
+  mixedNested = 'mixed-nested.md',
+  dirty = 'dirty.md'
+}
+
+export interface Fixture {
+  markdown: string
+  html: string
+  parsedDocument: Document
+}
+
+export type Fixtures = Record<keyof typeof FixtureFiles, Fixture>
+const DOMParser = new JSDOM().window.DOMParser
+
+/**
+ * Loads all fixtures
+ *
+ * @return All fixture files
+ */
+export function loadFixtures(): Fixtures {
+  return {
+    bullet: loadFixture(FixtureFiles.bullet),
+    ordered: loadFixture(FixtureFiles.ordered),
+    mixedNested: loadFixture(FixtureFiles.mixedNested),
+    dirty: loadFixture(FixtureFiles.dirty)
+  }
+}
+
+/**
+ * Loads a fixture file
+ *
+ * @param fileName The name of the fixture file
+ * @return the loaded fixture
+ */
+export function loadFixture(fileName: FixtureFiles): Fixture {
+  const mdParser = new MarkdownIt().use(taskLists)
```

Comment:
Is it a good idea to create a new markdownit for each test?

Reply:
It's not for every test. It's for every fixture. So per complete jest run it happens 4 times. imho that's okay.